Features/FindDataById/testPages/data/find_data_page.py

```diff
@@ -0,0 +1,173 @@
+
+import time
+from selenium.webdriver.common.by import By
+from Features.FindDataById.userInputs.user_inputs import UserData
+from common_utilities.selenium.base_page import BasePage
+
+
+""""Contains test page elements and functions related to the Lookup Table module"""
+
+class FindIdPage(BasePage):
+
+    def __init__(self, driver):
+        super().__init__(driver)
+
+        #FindDataById
+        self.find_data_by_id= (By.LINK_TEXT, "Find Data by ID")
+        self.find_id= "//*[@placeholder='{} ID']"
+        self.find_button = "//fieldset[@id='find-{}']//button[@type='button'][normalize-space()='Find']"
+        self.error_message ="//div[contains(text(),'Could not find {}')]"
+        self.export_link = "//a[normalize-space()='{} data export']"
+        self.export_page ="//h1[normalize-space()='Export {} Data']"
+        self.case_id_found = (By.XPATH,"//a[normalize-space()='View']")
+        self.view = "//fieldset[@id='{}']//a[contains(text(),'View')]"
+        self.location_name = (By.XPATH,"//p[@class='lead' and contains(text(),' Test Location [DO NOT DELETE!!!]')]")
+
+        # Find_Data_By_Id_Form_Level
+        self.case_change = (By.XPATH, "//a[normalize-space()='Case Changes']")
+        self.form_metadata = (By.XPATH, "//a[normalize-space()='Form Metadata']")
+        self.form_properties = (By.XPATH, "//a[normalize-space()='Form Properties']")
+        self.properties = "//a[normalize-space()='{}']"
+        self.id_values = "//dt[@title='{}']//following-sibling::dd[1]"
+        self.view_button = (By.XPATH, "//a[normalize-space()='View']")
+
+
+    def find_data_by_id_page_ui(self):
+        self.wait_to_click(self.find_data_by_id)
+        self.is_present_and_displayed((By.XPATH,self.find_id.format('Case')))
```

Comment:
Use the assert directly to line 37 and 38. Even though you are using is_present_and_displayed but not recording its return value in any condition so better perform the assertions directly in these 2 lines and remove assert True.

Reply:
Updated the necessary steps.